kratos/testing/tester.cpp

```diff
@@ -228,8 +228,7 @@ namespace Kratos
 		{
 			// creating the regex pattern replacing * with ".*"
 
-#if defined(__GNUC__) &&( __GNUC__ < 4 || \
-              (__GNUC__ == 4 && (__GNUC_MINOR__ < 9))) 
+#if __cplusplus <= 199711L
```

Comment:
Looks like they still define this as 199711L.

The question is, since we now have C+11 as a requisite can't we just compile the regex code as is?